src/KafkaFlow.Serializer.JsonCore/JsonCoreDeserializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace KafkaFlow.Serializer
```

Comment:
Honest doubt: This shouldn't be `KafkaFlow.Deserializer`?

Reply:
The same applies for other Deserialziers (avro and protobuf).

Reply:
@joelfoliveira, @ailtonguitar, @simaoribeiro, @ruiqbarbosa, @erik-catalao, @JoaoRodriguesGithub, @filipeesch any thoughts about that?

Reply:
There is no standard name for an abstraction that includes both Serializer and Deserializer implementations (the same way for Compressor and Decompressor), so I think we should keep the Serializer name as is the closest we have for a "standard" naming in the industry

Reply:
I agree with @simaoribeiro

Reply:
I agree with @simaoribeiro. Maybe in the v4 we can change to Serialization/Compression as suggested by @JoaoRodriguesGithub